### ISSUE TITLE
add manual gain control

### ISSUE THREAD

**Issue by jpggithub** [dkim0419/SoundRecorder#32]
Can you add a manual gain control?
Anyway, thanks for this open source software!

**Comment by fabianrost84** [comment]
Do you mean microphone gain during recording?

**Comment by jpggithub** [comment]
Yes. Sorry, it was not clear.

**Comment by fabianrost84** [comment]
Yes, I agree. It would be great for me as well, especially as I usually record very different sound levels. However, currently audio is recorded using the [`MediaRecorder`](https://developer.android.com/reference/android/media/MediaRecorder.html). And as far as I understand this won't allow for gain control. One would have to shift to [`AudioRecord`](https://developer.android.com/reference/android/media/AudioRecord.html) to do this.
